src/lib.rs

```diff
@@ -94,59 +113,129 @@ fn write_mod_cmd<W: Write>(config: &ModConfig, ctx: &ConfigContext, mut writer:
+                            if !matches!(res, Some(st) if st.success()) {
+                                log.error(format!("Task \"{}\" failed when run.", task.command));
+                                if task.terminate_on_errors {
+                                    std::process::exit(0);
+                                }
+                            }
+                        } else {
+                            log.error(format!("Task \"{}\" in invalid location.", task.command));
+                            if task.terminate_on_errors {
+                                std::process::exit(0);
+                            }
+                        }
+                    } else {
+                        log.error(format!(
+                            "Task \"{}\" not found. ({:?})",
+                            task.command,
+                            dunce::simplified(cmd_path.as_path())
+                        ));
+                        if task.terminate_on_errors {
+                            std::process::exit(0);
+                        }
+                    }
                 }
             }
         }
     }
     Ok(())
 }
 
-fn get_game_path() -> Result<PathBuf> {
-    let exe = std::env::current_exe()?;
-    let path = exe.parent().unwrap().parent().unwrap().parent().unwrap();
-    Ok(path.to_path_buf())
-}
-
-fn get_configs() -> Result<Vec<ModConfig>> {
-    let path = get_game_path()?.join("r6").join("config").join("cybercmd");
+pub fn get_configs(log: &mut Logger) -> Result<Vec<ModConfig>> {
+    let path = paths::get_game_path()?
+        .join("r6")
+        .join("config")
+        .join("cybercmd");
     let mut configs = vec![];
 
-    for entry in std::fs::read_dir(path)? {
+    log.debug("Getting configs.");
+
+    for entry in fs::read_dir(path)? {
         let entry = entry?;
         if entry.path().extension() == Some(OsStr::new("toml")) {
-            let contents = std::fs::read_to_string(entry.path())?;
-            configs.push(toml::from_str(&contents)?);
+            log.debug(format!(
+                "Loading: {}",
+                dunce::simplified(&entry.path()).display()
+            ));
+            let contents = fs::read_to_string(entry.path())?;
+            match toml::from_str(&contents) {
+                Ok(config) => configs.push(config),
+                Err(error) => log.error(format!(
+                    "In {} ({}): {}",
+                    dunce::simplified(&entry.path()).display(),
+                    match error.span() {
+                        Some(val) => format!("{:?}", val),
+                        None => "".to_string(),
+                    },
+                    error.message()
+                )),
+            };
         }
     }
     Ok(configs)
 }
 
+pub fn log_dir() -> PathBuf {
+    let log_dir = (match paths::get_game_path() {
+        Ok(game_path) => game_path,
+        Err(e) => panic!("cybercmd cannot get game path: {}", e),
+    })
```

Comment:
I think this is equivalent to
```suggestion
    let log_dir = paths::get_game_path().expect("cybercmd cannot get game path")
```
(expect includes formatted error by default too)